### ISSUE TITLE
chore(gatsby-cli): update prompts package to fix Node 8 error

### ISSUE THREAD

**Issue by tonystaark** [gatsbyjs/gatsby#14139]
Edit by @m-allanson: See [my comment below](https://github.com/gatsbyjs/gatsby/issues/14139#issuecomment-494477360) on how to resolve this issue.

## Description

gastsby new projectName not working and throw error UNHANDLED EXCEPTION
### Steps to reproduce

`npm i -g gatsby-cli`
`gatsby new blogName`

### Expected result
Should create a directory with name blogName

### Actual result

```
Error UNHANDLED EXCEPTION


  Error: /Users/TonyNg/.nvm/versions/node/v8.1.2/lib/node_modules/gatsby-cli/nod  e_modules/prompts/lib/index.js:35
        question[key] = typeof value === 'function' ? await value(answer, { ...a  nswers }, question) : value;
                                                                            ^^^
  SyntaxError: Unexpected token ...
  
  - module.js:11 require
    internal/module.js:11:18
  
  - index.js:4 Object.<anonymous>
    [lib]/[gatsby-cli]/[prompts]/index.js:4:7
```

### Environment

No project directory created so unable to run `gatsby info --clipboard`

**Comment by jonniebigodes** [comment]
@tonystaark just finished creating a new Gatsby website with your description, the exact same name and no issues whatsoever, could you provide a little more context? 
- What operating system are you using?
- Could you please remove the cli globally and re-install it and check if it solves the issue, probably something went wrong with the download and installation.

Feel free to provide feedback and sorry for the inconvenience

**Comment by KyleAMathews** [comment]
@tonystaark what version of node are you running?

**Comment by KyleAMathews** [comment]
You can run `gatsby info` in your site and paste the output here.

**Comment by tonystaark** [comment]
Hi I am running v8.1.2 for Node

After running gatsby info , I got 
```
 System:
    OS: macOS High Sierra 10.13.6
    CPU: (4) x64 Intel(R) Core(TM) i5-4258U CPU @ 2.40GHz
    Shell: 3.2.57 - /bin/bash
  Binaries:
    Node: 8.1.2 - ~/.nvm/versions/node/v8.1.2/bin/node
    Yarn: 1.16.0 - /usr/local/bin/yarn
    npm: 6.8.0 - ~/.nvm/versions/node/v8.1.2/bin/npm
  Languages:
    Python: 2.7.10 - /usr/bin/python
  Browsers:
    Chrome: 74.0.3729.157
    Firefox: 66.0.3
    Safari: 12.1.1
  npmGlobalPackages:
    gatsby-cli: 2.5.1
```

**Comment by jimcal** [comment]
I have similar issue with @tonystaark, here is my `gatsby info`.

```

  System:
    OS: macOS 10.14.4
    CPU: (4) x64 Intel(R) Core(TM) i5-5257U CPU @ 2.70GHz
    Shell: 5.3 - /bin/zsh
  Binaries:
    Node: 8.0.0 - ~/.nvm/versions/node/v8.0.0/bin/node
    Yarn: 1.15.2 - /usr/local/bin/yarn
    npm: 6.9.0 - ~/.nvm/versions/node/v8.0.0/bin/npm
  Languages:
    Python: 2.7.15 - /usr/local/bin/python
  Browsers:
    Chrome: 74.0.3729.157
    Firefox: 59.0.2
    Safari: 12.1
  npmGlobalPackages:
    gatsby-cli: 2.5.15
```

**Comment by lfalcao** [comment]
I had the same problem with node 8.0.0, using 10.15.3 as @jimcal said and worked... :taco:

**Comment by KyleAMathews** [comment]
It looks like the prompts package broke us. We should look into pinning it to an older version if we can.

**Comment by pieh** [comment]
Or swap to https://github.com/SBoudrias/Inquirer.js which does claim to support node 6+

**Comment by pieh** [comment]
Opened pull request in upstream package https://github.com/terkelg/prompts/pull/160 that will fix this error on Node >= 8.0 && < 8.6

**Comment by m-allanson** [comment]
Update gatsby-cli to prompts 2.0.5 once released.

**Comment by terkelg** [comment]
@m-allanson: A new [release](https://github.com/terkelg/prompts/releases/tag/v2.1.0) has been published

**Comment by m-allanson** [comment]
Super, thank you for the swift update @terkelg 🎉

**Comment by sidharthachatterjee** [comment]
Published in `gatsby-cli@2.6.2`
